staging/src/k8s.io/client-go/util/jsonpath/jsonpath.go

```diff
@@ -297,11 +297,11 @@ func (j *JSONPath) findFieldInValue(value *reflect.Value, node *FieldNode) (refl
 	var inlineValue *reflect.Value
 	for ix := 0; ix < t.NumField(); ix++ {
 		f := t.Field(ix)
-		jsonTag := f.Tag.Get("json")
-		parts := strings.Split(jsonTag, ",")
-		if len(parts) == 0 {
+		jsonTag, found := f.Tag.Lookup("json")
```

Comment:
Almost.  Strings.Split cannot return nil or an empty slice, so `len(parts) == 0` could never be true, so was clearly wrong.

I actually think that in many ways the ideal behaviour would be to recurse only if a json tag exists and is of the form `json:".*,inline"` but I'm worried in case making that change breaks something.  Also AFAICS the Go json library doesn't actually mention the existence of "inline" on a json tag, even though k8s uses it.

So I thought the best fix was to guarantee not to recurse if a json tag doesn't exist (in this case the original intention cannot have been to inline something), and to keep the behaviour identical in the case that the tag does exist (namely: consider inline/recurse if the first value is empty (`json:""`; `json:",foo"`; `json:","`, etc).